src/main/webapp/WEB-INF/tags/instructor/home/courseTable.tag

```diff
@@ -8,11 +8,13 @@
         <table class="table-responsive table table-striped table-bordered">
             <thead>
                 <tr>
-                    <th onclick="toggleSort(this);" class="button_sortname button-sort-none">
+                    <th class="button_sortname button-sort-none toggle-sort">
                         Session Name<span class="icon-sort unsorted"></span>
                     </th>
-                    <th onclick="toggleSort(this,instructorHomeDateComparator);" class="button_sortstartdate button-sort-none">Start Date<span class="icon-sort unsorted"></span></th>
-                    <th onclick="toggleSort(this,instructorHomeDateComparator);" class="button_sortenddate button-sort-none">End Date<span class="icon-sort unsorted"></span></th>
+                    <th class="button_sortstartdate button-sort-none toggle-sort"
+                        data-toggle-sort-comparator="sortDate">Start Date<span class="icon-sort unsorted"></span></th>
+                    <th  class="button_sortenddate button-sort-none toggle-sort"
+                         data-toggle-sort-comparator="sortDate">End Date<span class="icon-sort unsorted"></span></th>
```

Comment:
Perhaps you'd want to flip table after finding out why this would break something, but here goes: https://github.com/TEAMMATES/teammates/blob/master/src/main/webapp/js/common.es6#L440.